geti_sdk/data_models/shapes.py

```diff
@@ -119,7 +119,7 @@ def area(self) -> float:
         raise NotImplementedError
 
 
-@attr.s(auto_attribs=True)
+@attr.define(auto_attribs=True)
```

Comment:
Will `Rectangle` now be a slotted class or not? It doesn't have `slots=False`, but it does inherit from the non-slotted class `Shape`.

Same question for the other Shapes

Reply:
I think making them non-slotted makes sense. What do you think?